Mention adding Read for Review label to module guidelines (#873)
nf-core/nf-co.re: markdown/developers/adding_modules.md

Commit and push these changes to your local clone on GitHub, and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) on the `nf-core/modules` GitHub repo with the appropriate information.

> When you have created your pull request, please label it with the `Ready for Review` badge on the GitHub PR tab, and providing that everything adheres to nf-core guidelines we will endeavour to approve your pull request as soon as possible.

Reviewer: ```suggestion
When you are happy with your pull request, please label it with the `Ready for Review` badge on the GitHub PR tab, and providing that everything adheres to nf-core guidelines we will endeavour to approve your pull request as soon as possible.
```
